tree/treeplayer/src/TTreeReaderGenerator.cxx

```diff
@@ -828,8 +838,8 @@ R"CODE(//////////////////////////////////////////////////////////
       ofs <<
 R"CODE(//////////////////////////////////////////////////////////
 
-#ifndef )CODE" << fClassname << R"CODE(_h
-#define )CODE" << fClassname << R"CODE(_h
+#ifndef )CODE" << ROOT::Internal::GetCppName(fClassname) << R"CODE(_h
```

Comment:
Should `fClassname` be updated instead?

Reply:
See comment above about user readability / findability of generated files.

Reply:
Here too, we should distinguish the variable name between filename and classname.